storcli.py

```diff
@@ -98,8 +100,9 @@ def handle_megaraid_controller(response):
     add_metric('ports', baselabel, response['HwCfg']['Backend Port Count'])
     add_metric('scheduled_patrol_read', baselabel,
                int('hrs' in response['Scheduled Tasks']['Patrol Read Reoccurrence']))
-    for cvidx, cvinfo in enumerate(response['Cachevault_Info']):
-        add_metric('cv_temperature', baselabel + ',cvidx="' + str(cvidx) + '"', int(cvinfo['Temp'].replace('C','')))
+    if 'Cachevault_info' in response:
```

Comment:
There is a discrepancy in the case of `info` vs `Info` in the following line. Was that intentional?

Reply:
Good spot, no it wasn't!